zebra/zebra_pw.c

```diff
@@ -142,18 +142,6 @@ check_lsp (struct zebra_pw_t *pw)
       zlog_warn ("No rib found for PW %u at VPN %s", pw->pwid, pw->vpn_name);
       return 1;
     }
-  /* check labels for each nexthop in Route */
-  /*
-   * Need to ensure that there's a label binding for all nexthops.
-   * Otherwise, ECMP for this route could render the pseudowire unusable.
-   */
-  for (ALL_NEXTHOPS_RO(rib->nexthop, nexthop, tnexthop, recursing))
-    if (!nexthop->nh_label || nexthop->nh_label->num_labels == 0)
-      {
-        zlog_warn ("No label found in rib for PW %u at VPN %s", pw->pwid,
-                   pw->vpn_name);
-        return 1;
-      }
```

Comment:
Ya, you are right, good catch. I saw the other code (where it discards Implicit Null labels) but I didn't touch because I thought that it would be there for a reason and I was afraid of breaking something else.
I'll try removing it and restoring the check. If it works I'll add it to my final PR when I address all of your comments. Are you sure it's safe to install those implicit null labels? Why was that code put there then?
Thanks.